src/odemis/gui/model/__init__.py

```diff
@@ -595,6 +595,38 @@ def __init__(self, main):
         # VA for autofocus procedure mode
         self.autofocus_active = BooleanVA(False)
 
+
+class LocalizationGUIData(MicroscopyGUIData):
+    # TODO: update the doc string
+    """ Represent an interface used to only show the current data from the microscope.
+
+    It should be able to handle SEM-only, optical-only, SECOM and DELPHI systems.
+
+    """
+
+    def __init__(self, main):
+        assert main.microscope is not None
+        MicroscopyGUIData.__init__(self, main)
+
+        # Current tool selected (from the toolbar)
+        tools = {TOOL_NONE, TOOL_RULER}  # TOOL_ZOOM, TOOL_ROI}
+
+        # Update the tool selection with the new tool list
+        self.tool.choices = tools
+
+        # VA for autofocus procedure mode
+        self.autofocus_active = BooleanVA(False)
+
+        self.Zmin = model.FloatContinuous(
+            value=-10e-6, range=(-1000e-6, 0), unit="m")
+        self.Zmax = model.FloatContinuous(
+            value=10e-6, range=(0, 1000e-6), unit="m")
+        self.Zstep = model.FloatContinuous(
+            value=1e-6, range=(-100e-6, 100e-6), unit="m")
```

Comment:
The "tradition" is to name the VAs like "firstSecond". We already have "zPos". So we should name these attributes: zMin, zMax, zStep.